source/s3_meta_request.c

```diff
@@ -234,6 +236,14 @@ int aws_s3_meta_request_init_base(
 
     /* If the request's body is being passed in some other way, set that up.
      * (we checked earlier that the request body is not being passed multiple ways) */
+    if (options->receive_filepath.len > 0) {
```

Comment:
This is between the doc and the code. So now, the doc doesn't correlate with the code.